cocos/core/components/component-event-handler.ts

```diff
@@ -101,7 +101,6 @@ export class EventHandler {
      * 事件响应组件和函数所在节点
      */
     @serializable
-    @type(legacyCC.Node)
```

Comment:
How about add a comment to describe why we should not use `@type` decorator here and where the `type` information is registered ?

Reply:
ok, i would add comments

Reply:
`@type(Node)` introduce circular reference(_node_ -> _component_ -> _component-event-handler_ -> _node_) that's why our hack module _./component-event-handler.schema_ comes